ui/imodel-components-react/src/imodel-components-react/UiIModelComponents.json

```diff
@@ -133,6 +139,31 @@
     },
     "popupButton": {
       "setFormat": "Set Format"
+    },
+    "ratio-type": {
+      "n-to-one": {
+        "label": "N:1",
+        "description": "N to 1 ratio (e.g. N:1)"
+      },
+      "one-to-n": {
+        "label": "1:N",
+        "description": "1 to N ratio (e.g. 1:N)"
+      },
+      "use-greatest-common-divisor": {
+        "label": "Use GCD",
+        "description": "Simplifies ratio using the greatest common divisor (e.g. 0.3 → 3:10)"
+      },
+      "value-based": {
+        "label": "Value Based",
+        "description": "Scales the smaller value to 1 (e.g. 0.5 → 2:1, 2 → 1:2)"
+      },
+      "default": {
+        "description": "How properties are shown as a ratio."
+      }
+    },
+    "azimuthType": {
+      "baseTooltip": "The starting direction (base) from which azimuth angles are measured.",
```

Comment:
@rschili do you have a better explanation in mind? Not sure this is clear enough...